apiary.apib

```diff
@@ -398,11 +398,318 @@ MerkleDAG.
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Fri, 29 Jan 2016 20:40:32 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (Error)
+        - Message: 'incorrectly formatted key: '
+        - Code: 0
+
+    + Body
+
+        ```
+        {
+          "Message": "incorrectly formatted key: ",
+          "Code": 0
+        }
+        ```
+
++ Request With Key
+
+    #### curl
+
+        curl -i http://localhost:5001/api/v0/bitswap/unwant?arg=QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ
+
+    + Body
+
+        ```
+        curl -i http://localhost:5001/api/v0/bitswap/unwant?arg=QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ
+        ```
+
++ Response 200 (application/json)
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Fri, 29 Jan 2016 20:35:58 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Body
+
+        ```
+        ```
+
+## wantlist [GET /bitswap/wantlist{?peer}]
+Show blocks currently on the wantlist.
+
++ Parameters
+    + peer (Multihash, optional)
+
+        Specify which peer to show wantlist for. Default: self.
+
++ Request No Arguments
+
+    #### curl
+
+        curl -i http://localhost:5001/api/v0/bitswap/wantlist
+
+    + Body
+
+        ```
+        curl -i http://localhost:5001/api/v0/bitswap/wantlist
+        ```
+
++ Response 200
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Content-Type: application/json
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Fri, 29 Jan 2016 20:58:35 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes
+        + Keys: null (array) The keys
+
+    + Body
+
+        ```
+        {
+          "Keys": null
+        }
+        ```
+
++ Request Bad Argument
+
+    #### curl
+
+        curl -i http://localhost:5001/api/v0/bitswap/wantlist?peer=cat.jpg
+
+    + Body
+
+        ```
+        curl -i http://localhost:5001/api/v0/bitswap/wantlist?peer=cat.jpg
+        ```
+
++ Response 500
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Content-Type: application/json
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Fri, 29 Jan 2016 21:09:22 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (Error)
+        - Message: 'multihash too short. must be \u003e 3 bytes'
```

Comment:
It's the greater than sign, which is escaped because the content is application/json.